app.py

```diff
@@ -79,39 +73,33 @@ def get_data(file_id=None):
     :return: Returns a dictionary with the entries to be used when generating
     the facets and/or table data
     """
-    # Setup logging
     logger = app.log
+
     # Get all the parameters from the URL
-    logger.debug('Parameter file_id: {}'.format(file_id))
     if app.current_request.query_params is None:
         app.current_request.query_params = {}
     filters = app.current_request.query_params.get('filters', '{}')
-    logger.debug("Filters string is: {}".format(filters))
+
     try:
-        logger.info("Extracting the filter parameter from the request")
         filters = ast.literal_eval(filters)
         # Make the default pagination
-        logger.info("Creating pagination")
         pagination = _get_pagination(app.current_request)
-        logger.debug("Pagination: \n".format(json_pp(pagination)))
+
         # Handle <file_id> request form
         if file_id is not None:
-            logger.info("Handling single file id search")
-            filters['file']['fileId'] = {"is": [file_id]}
+            filters['entity_id'] = {"is": [file_id]}
```

Comment:
Leave the filters with 'file' for now. The way filters are to be organized, is:

```
{
   "facetType": {
        "facetName":{
            "is": ["attribute1", "attribute2", ...]
        },
        .
        .
        .
    }
}
```

Where facetType is a logical grouping of the types of facets (currently we only have facets of type "file"). In the future we will start separating them into more accurate logical groups ("files" facets, "donor" facets, etc). Please not that the facet types and the index entity types are two separate things. They may or may not overlap. See the ICGC portal for an example: https://dcc.icgc.org/search